recipes/xarray-sentinel/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xarray-sentinel" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-sentinel-{{ version }}.tar.gz
+  sha256: eed441553d52a3f07286d972e089c0c2be1e2506e6caa1d2697b5c5d0a503e75
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools_scm
+  run:
+    - attrs
+    - dask-core
+    - fsspec
+    - python >=3.7
```

Comment:
Missed this one but it should be the same version as the one in host.

Reply:
Whoops. I'll fix it in the feedstock.